src/components/screens/home-screen/home-screen.tsx

```diff
@@ -69,7 +75,7 @@ class HomeScreen extends React.Component<Props, State> {
         <AppLayout accessToken={ accessToken } keycloak={ keycloak }>
           <Grid container className={ classes.backgroundContainer }>
             <Typography className={ classes.title } variant="h2">
-              { strings.welcome.hello } { userName }
+              { this.greeting()  + ", " + userName  }
```

Comment:
```suggestion
              { `${ this.generateGreeting() }, ${ userName }`  }
```
You can use a string format like this + more descriptive method name